otwcode/otwarchive
# AO3-5707: Fix for legacy warning search

## Issue

https://otwarchive.atlassian.net/browse/AO3-5707

## Purpose

Branches off AO3-5613 (Warning class rename) to enable support for browser-bookmarked searches including warning tag ids. 

## Testing Instructions

* Copy the link for a search that includes a warning filter.
* Change archive_warning_ids to warning_ids.
* Go to that url and see if you get the same results.

## References

Should obviously be merged after #3526
